templates/compose/trailbase.yaml

```
@@ -0,0 +1,14 @@
+# documentation: https://trailbase.io/
+# slogan: A blazingly fast Rust/SQLite/Wasmtime app server with type-safe APIs
+# category: backend
+# tags: trailbase,backend,saas,mobile,api
+# logo: svgs/trailbase.svg
+# port: 4000
+
+services:
+  trailbase:
+    image: trailbase/trailbase:latest
```

Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.

```suggestion
    image: trailbase/trailbase:0.19.2 # Released on Oct 15, 2025 
```